R/make-request.R

```diff
@@ -0,0 +1,39 @@
+#' Make a Google API request
+#'
+#' Intended primarily for internal use in client packages that provide
+#' high-level wrappers for users. `make_request()` does very little: calls an
+#' HTTP method, only adding a user agent. Typically the input is created with
+#' [build_request()] and the output is processed with `process_response()`.
+#'
+#' @param x List. Holding the components for an HTTP request, presumably created
+#'   with [develop_request()] or [build_request()]. Must contain the a `method`
+#'   and `url`. If present, `body` and `token` are used.
+#' @param user_agent A user agent string, prepared by [httr::user_agent()].
```

Comment:
I think this should include a little advice: We recommending including the name of your package and its version in the user agent.